constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/utils/widget/MotionLabel.java

```diff
@@ -188,8 +188,7 @@ private void init(Context context, AttributeSet attrs) {
         setupPath();
     }
 
-
-    Bitmap blur(Bitmap bitmapOriginal, int factor) {
+Bitmap blur(Bitmap bitmapOriginal, int factor) {
```

Comment:
Indentation looks off here
```suggestion
    Bitmap blur(Bitmap bitmapOriginal, int factor) {
```